Add AZURE_TEST_NO_CLEANUP_ON_FAIL. Keep crime scene (resources) when a test case fails. (#71)
Azure/azure-python-devtools: HISTORY.rst

.. :changelog:

Release History
===============

1.3.0
++++++

> * Add AZURE_TEST_NO_CLEANUP_ON_FAIL. Do not delete resources when a test case fails when it is set.

Author: ```suggestion
* Add AZURE_TEST_NO_CLEANUP_ON_FAIL. Will not delete resources when a test case fails and the variable is set True.
```
